.gemspec

```diff
@@ -38,5 +38,39 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency(%q<hiera>, [">= 2.0", "< 4"])
   # PUP-7115 - return to a gem dependency in Puppet 5
   # s.add_runtime_dependency(%q<semantic_puppet>, ['>= 0.1.3', '< 2'])
+  # i18n support (gettext-setup and dependencies)
   s.add_runtime_dependency(%q<gettext-setup>, [">= 0.10", "< 1"])
+  s.add_runtime_dependency(%q<locale>, ["~> 2.1"])
+  s.add_runtime_dependency(%q<rake>, ["10.1.1"])
```

Comment:
@McdonaldSeanp I noticed that our shipped gem at https://rubygems.org/gems/puppet doesn't actually reference rake, though our `Gemfile` seems to indicate it's a dep.

I *think* that it's actually been expressed wrong in the `Gemfile` and should probably be labeled as a development dependency here instead.  It doesn't really matter to what we ship as those gems are generated by build tooling / automation and this file is ignored, but might be worth correcting here.